extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/NettyConfigGroup.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.micrometer.runtime.config;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Build / static runtime config for Netty Binders
+ */
+@ConfigGroup
+public class NettyConfigGroup implements MicrometerConfig.CapabilityEnabled {
+    /**
+     * Netty metrics support.
+     * <p>
+     * Support for Netty metrics will be enabled if Micrometer support is enabled,
+     * the Netty allocator classes are on the classpath
+     * and either this value is true, or this value is unset and
+     * {@code quarkus.micrometer.binder-enabled-default} is true.
+     */
+    @ConfigItem
+    public Optional<Boolean> enabled;
+
+    @Override
+    public Optional<Boolean> getEnabled() {
```

Comment:
I wonder if we should add some sort of documentation to this cool feature.

Reply:
Ah, good catch -- yeah, we should definitely add something.
I'll check what we currently have, and add something similar for this / Netty.